lib/sticky-session.js

```diff
@@ -244,7 +248,14 @@ StickyAgent.prototype.listener = function(msg, socket) {
           * If the header was not found return, probably unwanted behavior.
           */
         if (searchPos === -1) {
-          return;
+          if (self.ignoreMissingHeader) {
+            process.send(
```

Comment:
If you don't return in this block process.send would be executed twice, you probably should though.